### ISSUE TITLE
Forms in the plugin are inaccessible to screen readers

### ISSUE THREAD

**Issue by NeillM** [marceloschmitt/moodle-block_analytics_graphs#68]
There appear to be several forms in the plugin that are not accessible to screen readers because they do not associate the descriptions of fields with labels,

For example [in the e-mail form a screen reader will not be able to associate the Subject text with the subject input](https://github.com/marceloschmitt/moodle-block_analytics_graphs/blob/4.2/javascriptfunctions.php#L94)

[Web Content Accessibility Guidelines (WCAG) guide for form labels](https://www.w3.org/WAI/tutorials/forms/labels/)

**Comment by marceloschmitt** [comment]
Neil

I realized you have opened many issues. I will be able to look at them next week. Thanks a lot.

Marcelo

**Comment by NeillM** [comment]
Thanks Marcelo,

I thought it was best to raise several focused issues, rather than trying to combine them all into one big issue.

**Comment by marceloschmitt** [comment]
Created branch 4.3 for the corrections. Testing. I would appreciate if you test it.

**Comment by NeillM** [comment]
The forms look good for screen readers now.
